### ISSUE TITLE
Output folding

### ISSUE THREAD

**Issue by varemenos** [zeit/hyper#1093]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: OSX El Capitan (10.11.6)
- **Hyper.app version**: Version 0.8.3
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/varemenos/1dee1bd8dc17bc12221fc2bf1292dd48
- **Relevent information from devtools** _(CMD+SHIFT+I on Mac OS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: Is Vanilla

Would anyone be interested in implementing output folding like the one in [Final Term](http://worldwidemann.com/content/images/2015/03/finalterm.gif)?

**Comment by albertz** [comment]
This is supported in [DomTerm](https://domterm.org/) now ([details](https://github.com/PerBothner/DomTerm/issues/54)).
I really would like to see this also supported in Hyper.
I collected some overview of existing implementations [here](https://stackoverflow.com/questions/52812618/ansi-escape-sequence-for-collapsing-folding-text-maybe-hierarchically).

**Comment by j-f1** [comment]
Hyper uses xterm though.

**Comment by albertz** [comment]
Yes I know. But where is the problem with that? Or are you suggestion that:
* Hyper should use DomTerm instead of xterm.js?
* We should make this feature request upstream for xterm.js?

**Comment by jodinathan** [comment]
this would be very nice indeed!
